changelog/1208.breaking.rst

```diff
@@ -0,0 +1,4 @@
+Reorganization of `~plasmapy.dispersion.two_fluid_dispersion_solution`,
+The analytical solution file has been moved to a new folder inside 
+PlasmaP/plasmapy/dispersion/ called `analytical` and the file it self has been 
+renamed from two_fluid_dispersion_solution.py to two_fluid.py
```

Comment:
break this into two change logs

`1208.feature.rst`

```rst
Restructuring of the `plasmapy.dispersion` subpackage by creating the
`~plasmapy.dispersion.analytical` subpackage to contain functionality
related to analytical dispersion solutions.
```

`1208.breaking.rst`
```rst
Renamed file ``two_fluid_dispersion.py`` to ``two_fluid.py`` and moved into
the ``plasmapy.dispersion.analytical`` subpackage.  The contained function
``two_fluid_dispersion_solutions()`` was also renamed to
`~plasmapy.dispersion.analytical.two_fluid.two_fluid`.
```